app/src/components/market/common/outcome_table/index.tsx

```diff
@@ -22,6 +21,7 @@ interface Props {
   payouts?: Maybe<number[]>
   probabilities: number[]
   withWinningOutcome?: boolean
+  showSharesAndPriceChange?: boolean
```

Comment:
This should be called `showPriceChange`, or the change of shares should also be shown.